src/templates/app/config/api.yml.temp

```diff
@@ -62,7 +63,7 @@ services:
   api.authoriser.public:
     class: Lexide\LazyBoy\Security\PublicRouteAuthoriser
     tags:
-      - "api.authorisers"
+      api.authorisers: 10
```

Comment:
what's the context of 10?

Reply:
It's for ordering the authorisers. The public authoriser ought to be run early, but giving it 10 allows for something to be added before it, should that be required